paddle/fluid/framework/op_info.h

```diff
@@ -65,8 +65,14 @@ struct OpInfo {
   }
 
   const GradOpMakerFN& GradOpMaker() const {
-    PADDLE_ENFORCE_NOT_NULL(grad_op_maker_,
-                            "Operator GradOpMaker has not been registered.");
+    std::string type = Proto().type();
+    PADDLE_ENFORCE_NOT_NULL(
+        grad_op_maker_,
+        "Operator %s's GradOpMaker has not been "
+        "registered.\nPlease check whether %s_op has "
+        "grad_op.\nIf not, please set stop_gradient to True "
+        "for its input and output variables using var.step_gradient=True.",
```

Comment:
var.step_gradient=True => var.stop_gradient=True ?

Reply:
Sorry for typo. Thanks~